english/find-exit/leave.md

```diff
@@ -31,3 +31,5 @@ The forest turns dark. I don't see the ice-cold water and fall into it.
 [Invite some friends over to your room](../invite-friends/friends.md)
 
 [shoot a flare to wave down the chococopter](chococopter/chococopter.md)
+
+[Wave at the chococopter](../flight/flight.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.